docs/release-notes/Percona-Everest-1.4.0-(2024-12-15).md

```diff
@@ -0,0 +1,170 @@
+
+- [EVEREST-1683](https://perconadev.atlassian.net/browse/EVEREST-1683): The **Backups** on the **Overview** page are organized in descending order, making it easier to find your most recent backups by their start date and time.
+
+- [EVEREST-1686](https://perconadev.atlassian.net/browse/EVEREST-1686): We've adopted a 24-hour time format for our backups and restores to eliminate any potential confusion and ensure consistency across Percona Everest.
+
+- [EVEREST-1687](https://perconadev.atlassian.net/browse/EVEREST-1687): The label for the upgrade CRD button has been shortened to improve readability.  
+
+- [EVEREST-1701](https://perconadev.atlassian.net/browse/EVEREST-1701): Starting with Percona Everest 1.4.0, when configuring RBAC policies, the resource name for `database-cluster-backups` now corresponds to the database name instead of the backup name. This change allows for a more intuitive configuration of permissions for backups at the database level.
+
+
+- [EVEREST-1702](https://perconadev.atlassian.net/browse/EVEREST-1702): Starting with Percona Everest 1.4.0, when configuring RBAC policies, the resource name for `database-cluster-restores` now corresponds to the database name instead of the restore name. This change allows for a more intuitive configuration of permissions for backups at the database level.
+
+## Bugs
+
+- [EVEREST-1688](https://perconadev.atlassian.net/browse/EVEREST-1688): If a user changed the value for the number of shards and then scrolled down, the number of shards would unexpectedly increase. Conversely, if they scrolled up, the number of shards would decrease. The value did not remain constant, and similar behavior was observed in other fields as well. This issue has been resolved now, and the values for the fields remain constant.
+
+- [EVEREST-1187](https://perconadev.atlassian.net/browse/EVEREST-1187): We’ve resolved the issue that prevented Point-In-Time Recovery (PITR) from being enabled for the PostgreSQL database after setting up backup schedules on the **Backups** page.
+
+- [EVEREST-1235](https://perconadev.atlassian.net/browse/EVEREST-1235), [EVEREST-1254](https://perconadev.atlassian.net/browse/EVEREST-1254): `everestctl` now provides relevant error messages when using the `install` command, helping you identify any issues that occurred during the installation process. For instance, if the kubeconfig is unavailable, the cluster configuration is incorrect, or the cluster cannot be connected to, `everestctl` will display an appropriate error message.
+
+- [EVEREST-1254](https://perconadev.atlassian.net/browse/EVEREST-1254): During the uninstallation of Percona Everest, an unusual error code appeared at the end of the process. The issue has been resolved now.
+
+- [EVEREST-1301](https://perconadev.atlassian.net/browse/EVEREST-1301) \[UI\] Mongodb backup schedule can't be created if a schedule with a different backup storage is created in db edit
+
```

Comment:
I think we should remove this because since the editing wizard has been removed this bug no longer applies
```suggestion
```